rtos/Mutex.h

```diff
@@ -65,6 +65,9 @@ typedef mbed::ScopedLock<Mutex> ScopedMutexLock;
  @note
  Memory considerations: The mutex control structures are created on the current thread's stack, both for the Mbed OS
  and underlying RTOS objects (static or dynamic RTOS memory pools are not being used).
+
+ @note
```

Comment:
some are two lines, other are one line comment. is there a reason for the difference?

Reply:
No, I have made a new note follows the previous note format on every file.

Reply:
👍 makes sense!

Reply:
In bare metal, the Mutex class is a dummy implementation because the Mutex class is used to synchronize the execution of threads. So lock() and unlock() are no-ops.
For the purpose of this documentation, the API is supported.